libs/stringtable/src/lib.rs

```diff
@@ -43,51 +46,62 @@ static ALL_LANGUAGES: [&str; 25] = [
     "Danish",
 ];
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug)]
 pub struct Project {
-    #[serde(rename = "@name")]
-    name: String,
-    #[serde(rename = "Package")]
-    packages: Vec<Package>,
-
-    #[serde(skip)]
-    meta_comments: Vec<(String, String, Option<String>)>,
+    inner: InnerProject,
+    path: WorkspacePath,
+    keys: HashMap<String, Vec<Position>>,
```

Comment:
hashmaps have random order
so I think this is why the `insta` tests are failing?

Reply:
hoping 0de59759022443bcd0b0a3d3690a265d781423f3 makes sense
could also zero out `keys` as it really isn't the focus of the sort testing